app/grandchallenge/reader_studies/models.py

```diff
@@ -1176,6 +1191,34 @@ def assign_permissions(self):
         )
 
     def clean(self):
+        super().clean()
+        self._clean_answer_widget()
+        self._clean_answer_type()
+        self._clean_interface()
+
+    def _clean_answer_widget(self):
```

Comment:
I'm no longer sure whether this is the best place for these validations. We need to do those on the forms `clean` method before actually creating an instance of an `AnswerWidget`. The benefit of doing it here was that this also covers the admin forms, but I don't think we ever really create or edit questions in the admin, do we? If not, I think it simplifies things if we move these checks to the form's clean method.

Reply:
Having the validation in one place on the model is a good thing, not sure if it is possible.

Reply:
Yes, well, we could call `_clean_answer_widget` on the form's instance in the clean method on the form, but that does require setting the attributes on the instance before doing so (otherwise, `answer_widget` and `interface` and `required` will still be the old values and not those entered in the form). That feels a bit hacky and strange to me, so I currently tend to moving these checks to the form instead.

Reply:
I have removed the `_clean_answer_widget` now and will add these checks on the form in another PR.